### DISCUSSION TITLE
中州韵助手程序端已兼容雾凇拼音

### DISCUSSION THREAD

**Discussion by yanhuacuo** [rime/home#1568] [General]
![2024-05-24_113323](https://github.com/rime/home/assets/3107994/d91f2ea5-e017-454d-bd9a-b37779cfbcee)

![2024-05-24_113337](https://github.com/rime/home/assets/3107994/58456f36-4940-4b41-8656-e732b0414b0a)

![2024-05-24_113346](https://github.com/rime/home/assets/3107994/50123532-ee20-4abd-856b-b3c39622e4cf)

![2024-05-24_113354](https://github.com/rime/home/assets/3107994/3f0d2bc7-3dd4-48fe-9efb-9623460b3ee1)

![2024-05-24_113405](https://github.com/rime/home/assets/3107994/fb699675-4447-42a1-a137-e4920a6f1586)

![2024-05-24_113411](https://github.com/rime/home/assets/3107994/b69389b5-42dd-48ff-814c-7e6266fc08ba)

![2024-05-24_113433](https://github.com/rime/home/assets/3107994/4939ea60-714f-4862-824a-865317621d4c)

![2024-05-24_113441](https://github.com/rime/home/assets/3107994/ce5fdcfc-175c-4125-a1cc-a4bec59a4686)

![2024-05-24_113451](https://github.com/rime/home/assets/3107994/b3cf4cc1-dcdc-4628-b34f-1cd666280e81)

![2024-05-24_113700](https://github.com/rime/home/assets/3107994/698f5a0c-ad77-4c97-88f0-d9335eea32c6)

**Comment by LEOYoon-Tsaw** [comment]
此助手需要特殊格式，與rime不完全兼容，不建議使用
